docs/source/guide/index.rst

```diff
@@ -217,19 +216,14 @@ Step 2: Load the Online Key
 
     The RSTUF Worker uses this key during the process of managing the metadata.
 
-    Note: the ceremony process won't show any password or key content.
+    Note: It requires the public key information (key id/public hash) only.
```

Comment:
Can you add a sentence saying how to get this information?
Something similar to what you did when asking if the key should be public or private:
```suggestion
    Note: It requires the public key information (key id/public hash) only.
    Tip: "rstuf key info:" retrieves the public information
```